src/Sulu/Component/Content/Mapper/ContentMapper.php

```diff
@@ -1333,7 +1333,7 @@ public function loadByNode(
             false
         );
 
-        $availableLocalization = $this->getShadowLocale($contentNode, $availableLocalization);
+        $availableLocalization = $this->getResolvedLocale($contentNode, $availableLocalization);
```

Comment:
Should we also call this variable `$resolvedLocalization` then? It's not like it is the only one available..